### ISSUE TITLE
Armor stands do not react to gravity.

### ISSUE THREAD

**Issue by mibby** [pl3xgaming/Purpur#10]
**Describe the bug**
Armor Stands do not react to gravity.

**To Reproduce**
Place down an armor stand.

**Screenshots or Videos**
![link](https://i.imgur.com/L7cvR9m.png)

**Plugins**
``Plugins (81): AdvancedPortals, ArmorStandEditor, ArtMap, BannerBoard*, BannerMaker, BetterRTP*, BKCommonLib, BlockParticles, BossShopPro, BottledExp, BungeeTabListPlus*, ButtonWarp*, CasinoSlots, ChatControl, ChatReaction, Citizens, CitizensCMD, ColoredAnvils, CraftBook, CrateReloaded, CreativeItemControl, CreativeNbtControl, DailyBonus, DeluxeTags*, DiceRoller*, DiscordSRV, dynmap*, EchoPet, EmojiChat*, EnjinMinecraftPlugin, Essentials, EssentialsAntiBuild, EssentialsSpawn, ExtraContexts*, GravityTubes, HeadDatabase*, HolographicDisplays, InventoryFull*, ItemJoin, JumpPads*, LibsDisguises, LightCleaner, LimitPillagers, LiteBans, LogBlock, LuckPerms, LWC, Magic, mcMMO, Multiverse-Core, Multiverse-NetherPortals, NerfPhantoms, PerWorldInventory, PetMaster, PlaceholderAPI, Plan, PlayerParticles, PlayerVaults, PlayMoreSounds, Plot2Dynmap, PlotSquared, ProtocolLib, pvparena, RFChairs, ServerSigns, ShopChest, ShowItem, TAB, TitleManager, TokenManager, UltraBar, VanishNoPacket, Vault, VoidGenerator, VoidSpawn, Votifier, VotingPlugin, WGNoFlyZone*, WorldBorder, WorldEdit, WorldGuard``

**Version**
``This server is running Purpur version git-Purpur-370 (MC: 1.15.1) (Implementing API version 1.15.1-R0.1-SNAPSHOT)``

**Additional context**
Armor Stands reacted to gravity perfectly fine in purpur 1.14 with the same plugins. I also tried removing ArmorStandEditor to see if that was the culprit -- but it wasn't.

**Comment by Puremin0rez** [comment]
Verify that "armor-stands-tick" is set to true in paper.yml

**Comment by mibby** [comment]
Ah, I actually had that set to false since ArmorStands that were under water for decoration ticked quite often. When did it change that they need to tick for gravity to function? Use to work fine with that set to false.

I'll re-test later today with it enabled to verify.

**Comment by Puremin0rez** [comment]
The current behavior has been like that since around October 2019, and was implemented with this PR: https://github.com/PaperMC/Paper/pull/2608

It was changed because changing that option actually did nothing for most servers - the toggle only set the state of armor stands ticking for fresh ones spawned since the changing of that option, whereas the PR has changed it so the option now makes all armor stands regardless of when they were spawned no longer tick.

They've always needed to tick for gravity to affect them, it's just that you having that set to false was probably placebo for a very long time and they were all still indeed ticking. Now the setting actually properly works, and armor stands no longer tick if set to false.

**Comment by mibby** [comment]
Ah, I see. That's quite odd though since I toggled that behavior prior to the change and noticed a notable performance difference. I have quite a few hundred armor stands under water for decoration at my spawn so they are always loaded. Timings report indicated ArmorStand ticking was quite heavy in that area, even though they didn't take water damage. When I disabled ticking, the indication in the timings reports went away immediately and performance smoothed out -- while keeping gravity functional for newly placed stands. At least according to what I can remember in 1.13/1.14.

I'm only noticing and hearing reports of no gravity for stands recently, but it's most likely probably that change. I'll have to re-analyze performance of armor stands in water and see if it has improved much with 1.15. Thanks for the information though! I'll toggle it back enabled and see if stand gravity is function again tonight.
